src/app/core/interceptors/auth/auth.interceptor.ts

```diff
@@ -7,7 +7,7 @@ import { AuthService } from './../../../features/auth/services/auth/auth.service
 export const authInterceptor: HttpInterceptorFn = (req, next) => {
   const authService = inject(AuthService);
 
-  if (req.url.includes('/auth/public/')) {
+  if (req.url.includes('/auth/public/') || req.url.includes('/password/reset-request')||req.url.includes('/password/reset-password')) {
     return next(req);
   }
```

Comment:
if you have many urls check will you put all in your if statement?
I preffer you store all your urls in an array and check through the array rather
reach out if you need any help or clarification